src/tech/sw_hw.md

```diff
@@ -0,0 +1,68 @@
+# Software and Hardware Tracing
+
+Yorick has two tracing modes:
+
+ * Software tracing.
+ * Hardware tracing.
+
+## Software Tracing
+
+In software tracing mode, `ykrustc` inserts calls to a trace recorder at the
+beginning of each basic block. The arguments to the call identify the
+(statically known) location of the call-site. At runtime, when tracing is
+enabled the trace recorder stores each location into a trace buffer.
+
+### Further Reading
+
+ * A
```

Comment:
I'm not convinced having "A" on its own is a useful word wrap :)

Reply:
It's because the next line is a long link which can't (I don't think) be split.